connect/2.6/Dockerfile

```diff
@@ -6,7 +6,6 @@ LABEL maintainer="Debezium Community"
 ENV DEBEZIUM_VERSION="2.6.0.Alpha2" \
     MAVEN_REPO_CENTRAL="" \
     MAVEN_REPOS_ADDITIONAL="" \
-    MAVEN_DEP_DESTINATION=$KAFKA_CONNECT_PLUGINS_DIR \
```

Comment:
Sorry, I continue the previous discussion here in this line. In the essence of this PR, aren't we actually talking about this change here? We are now always defaulting to `KAFKA_CONNECT_PLUGINS_DIR` without any option to install to another destination directory, where previously we could customize.
@jpechane what about special dirs outside `KAFKA_CONNECT_PLUGINS_DIR` (which are also not lib) things like a scripting package, Apicurio, and other things that sometimes get symlinked into libs or the plugins dir based on env vars? Won't this be affected?

Reply:
Those are already handled by the `EXTERNAL_LIBS_DIR` configuration option, which is what apicurio does [here](https://github.com/razvanz/debezium-container-images/blob/main/connect-base/2.6/docker-maven-download.sh#L90) and OpenTelemetry does [here](https://github.com/razvanz/debezium-container-images/blob/main/connect-base/2.6/docker-maven-download.sh#L106).

Reply:
As every type of dependency is handled by a seperate function it makes sense to remove the common env var and use only the specific ones.